api/src/main/java/org/openmrs/module/pihcore/PihCoreActivator.java

```diff
@@ -115,7 +131,10 @@ public void stopped() {
     public void contextRefreshed() {
 
         setDispositionConfig(config);
+        setMetadataSharingResolvers();
```

Comment:
If this is all the programmatic MDS config we do, can we call this function `configureMetadataSharing()`?

Reply:
Actually, should this go in `MetadataSharingSetup`, and just be handled internally by that class?

Reply:
Agreed, this method should live in MetadataSharingSetup.  I'm going to keep it as a separate method within that class, because we want to call it from the contextRefreshed methoid, not the started method.